azkaban-common/src/main/java/azkaban/utils/DirectoryFlowLoader.java

```diff
@@ -60,8 +67,9 @@
   private Set<String> errors;
   private Set<String> duplicateJobs;
 
-  public DirectoryFlowLoader(Logger logger) {
+  public DirectoryFlowLoader(Props props, Logger logger) {
```

Comment:
Actually these sort of changes used to break reportal deployment, which involved replacing jars e.g. replacing azkaban-common jar but now azkaban-executor jar. But now things should be fixed as we have gradle build process for reportal clusters.